newIDE/app/src/ProjectManager/index.js

```diff
@@ -368,6 +385,14 @@ export default class ProjectManager extends React.Component<Props, State> {
     this._onProjectItemModified();
   };
 
+  _onOpenLayoutProperties = (layout: gdLayout) => {
+    this.setState({ lastOpenLayout: layout, layoutPropertiesDialogOpen: true });
+  };
+
+  _onOpenLayoutVariables = (layout: gdLayout) => {
+    this.setState({ lastOpenLayout: layout, layoutVariablesDialogOpen: true });
```

Comment:
Same :) 
You'll also need to accept the parameter as `?gdLayout`, so we can close the dialog by passing null.